apps/dolly-frontend/src/main/js/src/utils/hooks/useOrganisasjoner.tsx

```diff
@@ -157,17 +162,14 @@ export const useArbeidsforhold = (ident: string, harAaregBestilling: boolean, mi
 
 	const miljoer = miljoe ? [miljoe] : filteredEnvironments
 
-	const { data, error } = useSWR<Arbeidsforhold[], Error>(
+	const { data, error } = useSWR<Array<MiljoDataListe>, Error>(
 		[getArbeidsforholdUrl(miljoer), { 'Nav-Personident': ident }],
-		miljoer.length > 1 ? multiFetcherAny : fetcher,
+		multiFetcherFagsystemer,
```

Comment:
Husker det var en bug her hvis man bare hadde ett gyldig miljø hvor den returnerte feilmelding istedenfor, det trengs kanskje ikke lenger nå? 😅

Reply:
Jepp, den skal funke for bare ett miljø også 😎